src/thunderbots/software/test/evaluation/deflect_off_enemy_target.cpp

```diff
@@ -0,0 +1,48 @@
+//
+// Created by roark on 07/02/19.
+//
+
+#include "ai/hl/stp/evaluation/deflect_off_enemy_target.h"
+
+#include <gtest/gtest.h>
+
+#include "geom/util.h"
+
+
+TEST(DeflectOffEnemyTargetTest, deflect_off_enemy_target_test)
+{
+    auto test_ball = new Ball(Point(-1, 0), Vector(0, 0), Timestamp::fromMilliseconds(0));
```

Comment:
Please don't use raw pointers here. There's no need for these values to be pointers, so you can just make them normal variables (remove the `new` keyword).

This applies to everything in this file.